ui/components/MesheryPatterns.js

```diff
@@ -430,6 +441,10 @@ function MesheryPatterns({
     handleSubmit(link, "", "meshery_" + Math.floor(trueRandom() * 100), FILE_OPS.URL_UPLOAD);
     // console.log(link, "valid");
   }
+
+  function ghImportHandler(gh) {
+    handleSubmit(gh, "",  "meshery_" + Math.floor(trueRandom() * 100), FILE_OPS.GH_IMPORT);
```

Comment:
And the POST request endpoint is mentioned as `/api/experimental/pattern` [here](https://docs.google.com/document/d/1B2N78EdRiZF-yVo1-HY3syppwBBDumgMuYg6seD-AJ4/edit#). So I'm not sure if you can use the same `handleSubmit` as it makes request to `/api/pattern`

Reply:
yes we can use `/api/pattern` it's the new one, it was earlier `/api/experimental/pattern` confirmed.

Reply:
Great!